Content.Server/GameTicking/GameTickerCommands.cs

```diff
@@ -311,18 +311,43 @@ public void Execute(IConsoleShell shell, IPlayerSession player, string[] args)
                 return;
             }
 
-            if (args.Length != 2)
+            int mapId;
+            string mapName;
+
+            switch (args.Length)
             {
-                shell.SendText(player, Help);
-                return;
+                case 1:
+                    if (player.AttachedEntity == null)
+                    {
+                        shell.SendText(player, "The map name argument cannot be omitted if you have no entity.");
+                        return;
+                    }
+
+                    var mapManager = IoCManager.Resolve<IMapManager>();
+                    mapId = (int) mapManager.NextMapId();
+                    mapName = args[0];
+                    break;
+                case 2:
+                    if (!int.TryParse(args[0], out var id))
+                    {
+                        shell.SendText(player, $"{args[0]} is not a valid integer.");
+                        return;
+                    }
+
+                    mapId = id;
+                    mapName = args[1];
+                    break;
+                default:
+                    shell.SendText(player, Help);
+                    return;
             }
 
-            shell.ExecuteCommand(player, $"addmap {args[0]} false");
-            shell.ExecuteCommand(player, $"loadbp {args[0]} \"{CommandParsing.Escape(args[1])}\"");
+            shell.ExecuteCommand(player, $"addmap {mapId} false");
+            shell.ExecuteCommand(player, $"loadbp {mapId} \"{CommandParsing.Escape(mapName)}\"");
             shell.ExecuteCommand(player, $"aghost");
-            shell.ExecuteCommand(player, $"tp 0 0 {args[0]}");
+            shell.ExecuteCommand(player, $"tp 0 0 {mapId}");
 
-            shell.SendText(player, $"Created unloaded map from file {args[1]} with id {args[0]}. Use \"savebp 4 foo.yml\" to save it.");
+            shell.SendText(player, $"Created unloaded map from file {mapName} with id {mapId}. Use \"savebp 4 foo.yml\" to save it.");
```

Comment:
Maybe make it clear that savebp saves the grid? and maybe insert the current gridid there too